qos/cometbft/response_status.go

```diff
@@ -76,6 +67,12 @@ type responseToStatus struct {
 	// jsonRPCResponse stores the JSON-RPC response parsed from an endpoint's response bytes.
 	jsonRPCResponse jsonrpc.Response
 
+	// chainID stores the chain ID of the endpoint.
+	chainID string
+
+	// catchingUp indicates if the endpoint is catching up to the network.
```

Comment:
Please add link to the cometbft section covering this.

I know you have it in one other place, but I don't think it'll hurt.